sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1087,7 +1087,7 @@ public virtual Response SetAccessPolicy(IEnumerable<SignedIdentifier> tableAcl,
         /// <returns></returns>
```

Comment:
nit:  The empty tag offers no value.   You may want to consider filling it out or just removing it.